packages/frontend-2/components/dashboard/TutorialCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <NuxtLink class="w-full" to="tutorial.url" external>
```

Comment:
Please test that these links work, this is just a link w/ the URL "tutorial.url", it should be `:to=` not `to=`

Reply:
Also these should probably have target="_blank"

Reply:
@fabis94 Oops accidentally broke that in my last commit, fixed 😅